scala/scala.bzl

```diff
@@ -852,6 +860,8 @@ scala_test = rule(
   attrs={
       "main_class": attr.string(default="io.bazel.rulesscala.scala_test.Runner"),
       "suites": attr.string_list(),
+      "colors": attr.bool(default=False),
+      "full_stacktraces": attr.bool(default=False),
```

Comment:
I'm happy making both of these default to `True` and let people opt out, personally.

Reply:
Me too actually. We'd opt to enable these as our internal defaults either way, but I wasn't sure how much pushback I'd get from changing existing behavior if I did. Looks like not much :)